website/content/docs/operations/metrics-reference.mdx

```diff
@@ -93,13 +93,16 @@ be assumed to be 10 seconds when retrieving metrics using the above described
 | `nomad.runtime.alloc_bytes`                  | Memory utilization                                                                                                                                                                                                | # of bytes                     | Gauge   |
 | `nomad.runtime.heap_objects`                 | Number of objects on the heap. General memory pressure indicator                                                                                                                                                  | # of heap objects              | Gauge   |
 | `nomad.runtime.num_goroutines`               | Number of goroutines and general load pressure indicator                                                                                                                                                          | # of goroutines                | Gauge   |
 | `nomad.nomad.broker.total_pending`           | Evaluations that are pending until an existing evaluation for the same job completes                                                                                                                              | # of evaluations               | Gauge   |
 | `nomad.nomad.broker.total_ready`             | Number of evaluations ready to be processed                                                                                                                                                                       | # of evaluations               | Gauge   |
 | `nomad.nomad.broker.total_unacked`           | Evaluations dispatched for processing but incomplete                                                                                                                                                              | # of evaluations               | Gauge   |
+| `nomad.nomad.broker.wait_time`               | Time elapsed while the evaluation was ready to be processed and waiting to be dequeued                                                                                                                            | ns / Evaluation Wait           | Timer   |
+| `nomad.nomad.broker.process_time`            | Time elapsed while the evaluation was dequeued and finished processing                                                                                                                                            | ns / Evaluation Process        | Timer   |
+| `nomad.nomad.broker.response_time`           | Time elapsed from when the evaluation was last enqueued and finished processing                                                                                                                                   | ns / Evaluation Response       | Timer   |
```

Comment:
We're likely to get questions about this data when the numbers look weird after a leader election. I noted above that we should clear the maps, but we should note here that the metrics are only valid within a single term as well.

Reply:
corrected in [7e37065](https://github.com/hashicorp/nomad/pull/20329/commits/7e37065b3271b139179bec1ba01026f5e264617f)